frontend/src/metabase/search/components/SearchFilterModal/SearchFilterModalFooter.tsx

```diff
@@ -0,0 +1,28 @@
+import { t } from "ttag";
+import Button from "metabase/core/components/Button";
+import { Flex, Group } from "metabase/ui";
+import { CloseAllFiltersButton } from "metabase/search/components/SearchFilterModal/SearchFilterModalFooter.styled";
+
+type SearchFilterModalFooterProps = {
+  onApply: () => void;
+  onCancel: () => void;
+  onClear: () => void;
+};
+
+export const SearchFilterModalFooter = ({
+  onApply,
+  onCancel,
+  onClear,
+}: SearchFilterModalFooterProps) => {
+  return (
+    <Flex direction={"row"} justify={"space-between"} align={"center"}>
```

Comment:
You should use just a plain string when you can

```suggestion
    <Flex direction="row" justify="space-between" align="center">
```